stock_scheduler_assignation_horizon/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="res_config_settings_view_form_stock" model="ir.ui.view">
+        <field
+            name="name"
+        >res.config.settings.view.form.inherit.stock.logistics.workflow</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="stock.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div name="operations_setting_container" position="after">
+                <h2>Shipping Scheduler</h2>
+                <div
+                    class="row mt16 o_settings_container"
+                    name="shipping_schedule_setting_container"
+                >
+                    <div class="col-12 col-lg-6 o_setting_box">
```

Comment:
We're missing the multi-company icon informing these fields are company-specific

```xml
<span class="fa fa-lg fa-building-o" title="Values set here are company-specific." aria-label="Values set here are company-specific." groups="base.group_multi_company" role="img"/>
```